lib/assets/javascripts/react_ujs_mount.js

```diff
@@ -2,6 +2,31 @@
   // jQuery is optional. Use it to support legacy browsers.
   var $ = (typeof window.jQuery !== 'undefined') && window.jQuery;
 
+  // Get the constructor for a className
+  // Not a part of the public API
+  function getConstructor(className) {
```

Comment:
Am I reading the diff right? It looks like this function is in the source twice, but we only need the one that's attached to ReactRailsUJS, right?